slides/src/content/pipes.md

```diff
@@ -24,15 +24,15 @@
 
 ---
 <!-- .slide: id="pipes-adding-a-pipe" -->
-## Adding a Pipe
+## Using a Pipe
 
 - Put the name of the pipe inside `{{...}}`
 - Use vertical bar `|` as separator
 
 #####_src/app/to-do-list/to-do-list.component.html_
 ```html
 <ul>
-  <li *ngFor="let item of thingsToDo; let i = index" id="{{i}}">
+  <li *ngFor="let item of thingsToDo; let i = index" [id]="i">
```

Comment:
did you submit a PR for this to https://github.com/gvwilson/angular-todo-example ?